docs/administrator/security/certificate-configuration.md

```diff
@@ -0,0 +1,334 @@
+- `cfssl` ([Installation Guide](github.com/cloudflare/cfssl/cmd))
+- `cfssljson` ([Installation Guide](github.com/cloudflare/cfssl/cmd))
+
+
+
+> NOTE: The following examples are based on the installation method described in [Installation from Source](https://karmada.io/docs/installation/fromsource/#install). When using other installation methods, appropriate adaptation is required.
+
+### Karmada Certificates
+
+Karmada certificate storage path: Default to `${HOME}/.karmada`.
+
+The current certificates used by Karmada include:
+
+```shell
+$ tree
+.
+├── apiserver.crt
+├── apiserver.key
+├── ca-config.json
+├── ca.crt
+├── ca.key
+├── etcd-ca-config.json
+├── etcd-ca.crt
+├── etcd-ca.key
+├── etcd-client.crt
+├── etcd-client.key
+├── etcd-server.crt
+├── etcd-server.key
+├── front-proxy-ca-config.json
+├── front-proxy-ca.crt
+├── front-proxy-ca.key
+├── front-proxy-client.crt
+├── front-proxy-client.key
+├── karmada.crt
+└── karmada.key
+```
+
+#### **Karmada Certificate Overview**
+
+Certificates can be categorized into three sets based on the issuing CA:
+
+- Issued by ca
+
+  The certificates `apiserver` and `karmada` are both issued by the CA certificate `ca`. The key attributes of these certificates are as follows:
+
+  | Certificates | common name(CN)   | organization(og) | hosts                                                        |
+  | ------------ | ----------------- | ---------------- | ------------------------------------------------------------ |
+  | apiserver    | karmada-apiserver | /                | "*.etcd.karmada-system.svc.cluster.local" "*.karmada-system.svc.cluster.local" "*.karmada-system.svc" "localhost" "127.0.0.1" "${api server's ip from kubecomfig by context karmada host}" |
+  | karmada      | system:admin      | system:masters   | kubernetes.default.svc "*.etcd.karmada-system.svc.cluster.local" "*.karmada-system.svc.cluster.local" "*.karmada-system.svc" "localhost" "127.0.0.1" "${interpreter_webhook_example_service_external_ip_address}" |
+
+  The `${api server's ip from kubecomfig by context karmada host}` can be obtained by executing `./get_server_ip.sh ${HOST_CLUSTER_NAME}`.
+
+  ```shell
+  #!/usr/bin/env bash
+  # get_server_ip.sh
+  context_name=$1
+  cluster_name=$(kubectl config view --template='{{ range $_, $value := .contexts }}{{if eq $value.name '"\"${context_name}\""'}}{{$value.context.cluster}}{{end}}{{end}}')
+  apiserver_url=$(kubectl config view --template='{{range $_, $value := .clusters }}{{if eq $value.name '"\"${cluster_name}\""'}}{{$value.cluster.server}}{{end}}{{end}}')
+  echo "${apiserver_url}" | awk -F/ '{print $3}' | sed 's/:.*//'
+  ```
+
+  > PS: The `karmada api server config` in the `kubeconfig` file is composed of the `karmada` certificate.
+
+- Issued by etcd-ca
+
+  The certificates `etcd-server` and `etcd-client` are both issued by the CA certificate `etcd-ca`. The key attributes of these certificates are as follows:

```

Comment:
```suggestion
  The `etcd-server` and `etcd-client` certificates are both issued by `etcd-ca`. Below are the key attributes of these certificates:
```
Seems `etcd-ca` is a CA instead of a certificate.